kubernetes/resource_kubernetes_pod_security_policy.go

```diff
@@ -0,0 +1,494 @@
+package kubernetes
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
Will need rebase to master and change of this import to `github.com/hashicorp/terraform-plugin-sdk/helper/schema`